manim/scene/scene_file_writer.py

```diff
@@ -328,7 +328,7 @@ def begin_animation(self, allow_write=False, file_path=None):
             Whether or not to write to a video file.
         """
         if write_to_movie() and allow_write:
-            self.open_movie_pipe(file_path=file_path)
+            self.open_movie_pipe(file_path=file_path.name)
```

Comment:
I think this change was unnecessary.

```suggestion
            self.open_movie_pipe(file_path=file_path)
```

Reply:
Yeah, I tried fixing it that way but it didn't work